Homework8/Task1/MassiveElement.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace Homework8.Task1
+{
+    internal class MassiveElement
+    {
+        private static string? inputValue;
+
+        public static void ShowMassiveElement()
+        {
+            int[] massive = { 8, 7, 1, 4, 2 };
+            Console.WriteLine("Input index of element in massive:");
+            string? inputedValue = Console.ReadLine();
+            string? checkedValue = inputedValue.Equals(string.Empty) ? null : inputValue;
```

Comment:
You always get null here.
There is a typo in task, should be
string? checkedValue = inputedValue.Equals(string.Empty) ? null : inputedValue;

Reply:
Fixed